spec/controllers/queue_items_controller_spec.rb

```diff
@@ -119,20 +119,20 @@
       end
 
       it "redirects to the my queue page if authenticated" do
-        post :update_queue,
-             queue_items_data: {:list_orders => [], :ratings => []}
+        post :update_queue, params:
+          { queue_items_data: { list_orders: [5], ratings: [3] } }
         expect(response).to redirect_to queue_items_path
       end
 
       it "reorders the queue items" do
-        post :update_queue,
-             queue_items_data: { :list_orders => [3, 2, 1], :ratings => [] }
+        post :update_queue, params:
+          { queue_items_data: { list_orders: [3, 2, 1], ratings: [nil] } }
```

Comment:
@brandoncc  Hmm valid point. Initially, when these arrays were empty I was getting errors, so I threw a nil in there to get it to work. But, I wasn't thinking of what made sense, as a parameter at the time. I'll make sure change this thanks!